lib/ansible/plugins/callback/json.py

```diff
@@ -100,15 +100,24 @@ def v2_playbook_on_stats(self, stats):
             s = stats.summarize(h)
             summary[h] = s
 
-        custom_stats = {}
+        custom_stats = {
+            'global': {},
+            'hosts': {}
+        }
```

Comment:
This name spacing change is not backwards compatible and will cause existing uses of this plugin to fail unexpectedly.  I believe a different means will be required here.